docs/jwst/cube_build/arguments.rst

```diff
@@ -44,26 +46,25 @@ each band will be created.
   with the options above [band, channel, grating, filter] to fully control the type of IFU
   cubes to make.
 
-  - ``output_type = band`` is the default mode for calspec3 and creates IFU cubes containing only one band
+  - ``output_type = band`` creates IFU cubes containing only one band
     (channel/sub-channel for MIRI or grating/filter combination for NIRSpec).
 
   - ``output_type = channel`` creates a single IFU cube from each unique channel of MIRI data
-    (or just those channels set by the 'channel' option).
+    (or just those channels set by the 'channel' option). The is the default mode for the calspec3 pipeline for
```

Comment:
It would be good to make "calspec3" a reference/link to the calwebb_spec3 docs. And we should stick to the full official name, "calwebb_spec3", here in the docs.